README.md

```diff
@@ -1,6 +1,9 @@
 # Elixir Koans
 
-[![Build Status](https://travis-ci.org/elixirkoans/elixir-koans.svg?branch=master)](https://travis-ci.org/elixirkoans/elixir-koans)
+[![Build Status](https://travis-ci.org/krlsdu/elixir-koans.svg?branch=master)](https://travis-ci.org/krlsdu/elixir-koans)
```

Comment:
This and the other badges seem like a leftover. 
We might be interested to pull some of those things over to the elixirkoans organisation though 🤔

Reply:
My mistake fix on commit 
06187ba